third_party/libzmq.BUILD

```diff
@@ -0,0 +1,241 @@
+package(default_visibility = ["//visibility:public"])
+
+licenses(["notice"])  # LGPL license with exception for static linking (for libzmq)
+
+genrule(
+    name = "configure",
+    outs = ["platform.hpp"],
+    cmd = "cd external/libzmq && ./configure && cd ../..; cp external/libzmq/src/platform.hpp $@",
+)
```

Comment:
This might be problematic for the Windows/Bazel build... I wonder if this file can be used on Windows:

https://github.com/zeromq/libzmq/blob/master/builds/msvc/platform.hpp

(using the `if_windows()` macro to change the command to copy that file to `$@`)